docs/api_definitions/custom_plugin_python/custom_plugin_python.md

```diff
@@ -0,0 +1,73 @@
+## Custom Plugin example using python
+
+We assume that you already have a developed python plugin with the associated python file and its manifest in JSON.
+If not, you can start from the provided examples ("manifest.json" and "middleware.py").
+Furthermore, in this example, we are going to serve those as a bundle using a small http python server.
+
+
+Firstly to create the said bundle we need to run the following command:
+
+```
+IMAGETAG=v3.2.1
```

Comment:
This command won't run if your directory has no `manifest.json` attached to this PR. So, users should be in ` docs/api_definitions/custom_plugin_python` directory.